.github/workflows/build-docker-artifact.yaml

```diff
@@ -1,18 +1,26 @@
 name: "Build tt-metal docker artifact"
 
 on:
-  workflow_call:
   workflow_dispatch:
-
+  push:
+    branches:
+      - dimitri/8082-run-python-tests-on-wheel # Needs to be changed before the merge
```

Comment:
Change to main upon merge.

Reply:
Curious, I think there can be a disconnect if the user is not sure the docker image has finished building yet?

Example:
A user can push a commit up with changes for dockerfile
and immediately run post-commit. This would be that this particular post-commit run can start running before the new docker image is and ready for use?